### PR TITLE
Automatically loads files according to the PSR4

### PR DESCRIPTION
**Pull request by questocat**

### CODE REVIEW

File: tests/bootstrap.php [hunk 1]
```diff
@@ -7,4 +7,4 @@
  * @license   http://opensource.org/licenses/BSD-2-Clause Simplified BSD License
  */
 
-require_once __DIR__ . '/../autoload_register.php';
+require dirname(__DIR__).'/vendor/autoload.php';
```
Comment (DASPRiD) [inline tests/bootstrap.php]: There's no valid reason for this change.

Comment (emanci) [reply tests/bootstrap.php]: I may not agree. it offers greater ease of use and no need to regenerate the autoloader when you add classes, it's only a piece of advice.

Comment (DASPRiD) [reply tests/bootstrap.php]: Oh nevermind, I missed the last part of the line. I'd still not use `dirname()` though in favor of `__DIR__ . '/../vendor/autoload.php'`… Thinking aobut this, the bootstrap file can actually go away completely, just define the vendor/autoload.php als bootstrap in the phpunit.xml.

Comment (emanci) [reply tests/bootstrap.php]: Yes, i usually like to define the vendor/autoload.php als bootstrap in the phpunit.xml.

**Pull request closed without merging**